scripts/xjoin-config/elasticsearch-sink.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "xjoin.inventory.es",
+    "config": {
+      "type.name": "_doc",
+      "connector.class": "io.confluent.connect.elasticsearch.ElasticsearchSinkConnector",
+      "behavior.on.null.values": "delete",
+      "errors.log.include.messages": "true",
+      "transforms": "valueToKey, extractKey, expandJSON, expandPRSJSON, deleteIf, flattenList, flattenListString, flattenPRS, renameTopic",
+      "max.retries": "8",
+      "transforms.flattenList.keys": "namespace,key,value",
+      "transforms.extractKey.type": "org.apache.kafka.connect.transforms.ExtractField$Key",
+      "transforms.expandJSON.sourceFields": "tags",
+      "transforms.expandPRSJSON.sourceFields": "per_reporter_staleness",
+      "transforms.flattenPRS.rootKey": "reporter",
+      "transforms.flattenPRS.sourceField": "per_reporter_staleness",
+      "errors.log.enable": "true",
+      "transforms.deleteIf.type": "com.redhat.insights.deleteifsmt.DeleteIf$Value",
+      "transforms.flattenPRS.type": "com.redhat.insights.flattenlistsmt.FlattenList$Value",
+      "transforms.flattenListString.delimiterJoin": "/",
+      "transforms.flattenListString.type": "com.redhat.insights.flattenlistsmt.FlattenList$Value",
+      "max.in.flight.requests": "1",
+      "behavior.on.malformed.documents": "warn",
+      "transforms.renameTopic.regex": "xjoin.inventory.public.hosts",
+      "transforms.flattenListString.mode": "join",
+      "transforms.flattenList.outputField": "tags_structured",
+      "name": "xjoin.inventory.es",
+      "transforms.renameTopic.type": "org.apache.kafka.connect.transforms.RegexRouter",
+      "transforms.flattenListString.sourceField": "tags",
+      "auto.create.indices.at.start": "false",
+      "transforms.flattenListString.encode": "true",
+      "connection.password": "v4JAwf2I45yuX5rke0i4J532",
+      "key.ignore": "false",
+      "retry.backoff.ms": "100",
+      "max.buffered.records": "500",
+      "transforms.flattenPRS.mode": "object",
+      "transforms.valueToKey.type": "org.apache.kafka.connect.transforms.ValueToKey",
+      "transforms.deleteIf.field": "__deleted",
+      "transforms.flattenPRS.outputField": "per_reporter_staleness_flat",
+      "transforms.expandJSON.type": "com.redhat.insights.expandjsonsmt.ExpandJSON$Value",
+      "transforms.flattenList.type": "com.redhat.insights.flattenlistsmt.FlattenList$Value",
+      "transforms.extractKey.field": "id",
+      "transforms.expandPRSJSON.type": "com.redhat.insights.expandjsonsmt.ExpandJSON$Value",
+      "transforms.flattenList.sourceField": "tags",
+      "topics": "xjoin.inventory.public.hosts",
+      "batch.size": "100",
```

Comment:
The `batch.size` default is `2000` and should work with deployment?

Reply:
Done